### PR TITLE
chore: Angular 16 / nx 16 support

### PR DESCRIPTION
**Pull request by sevaru**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [x] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Only Angular 12 supported

## What is the new behaviour?

Support for Angular 16 / NX 16

### CODE REVIEW

**Review by sevaru — COMMENTED** [review]
Update libs peerDeps

File: libs/angular/project.json [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
   "prefix": "",
   "targets": {
     "build": {
-      "executor": "@nrwl/angular:package",
+      "executor": "@nx/angular:package",
       "options": {
         "tsConfig": "libs/angular/tsconfig.lib.json",
-        "project": "libs/angular/ng-package.json"
+        "project": "libs/angular/ng-package.json",
+        "updateBuildableProjectDepsInPackageJson": true
```
Comment (sevaru) [inline libs/angular/project.json:13]: is deprecated
https://nx.dev/nx-api/eslint-plugin/documents/dependency-checks

File: libs/angular/.babelrc [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["@nrwl/react/babel"],
+  "presets": ["@nx/react/babel"],
   "plugins": []
 }
```
Comment (sevaru) [inline libs/angular/.babelrc:4]: check do we need it at all

File: libs/angular/project.json (hunk 1 above)
Comment (sevaru) [reply libs/angular/project.json:13]: fixed

File: libs/angular/.babelrc (hunk 2 above)
Comment (sevaru) [reply libs/angular/.babelrc]: fixed, removed

**Review by MarsiBarsi — APPROVED** [review]

File: apps/host/src/index.html [hunk 3]
```diff
@@ -6,6 +6,17 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <script>
+      window.addEventListener('beforeunload', function (event) {
+        debugger;
```
[the remark this answers is not in the document]
Comment (sevaru) [reply apps/host/src/index.html:11]: i'll fix this thanks

File: apps/host/src/app/app.component.ts [hunk 4]
```diff
@@ -25,13 +25,13 @@ export class AppComponent {
...
-        status: TuiNotification.Success,
-      })
-      .subscribe();
+    // this.alertService
+    //   .open(event.type, {
+    //     hasIcon: false,
+    //     label: event.target?.name,
+    //     status: 'info'
+    //   })
+    //   .subscribe();
```
[the remark this answers is not in the document]
Comment (sevaru) [reply apps/host/src/app/app.component.ts]: fixed

**Review by MarsiBarsi — APPROVED** [review]

**Pull request merged**